numpy/core/src/multiarray/lowlevel_strided_loops.c.src

```diff
@@ -1707,10 +1712,22 @@ mapiter_@name@(PyArrayMapIterObject *mit)
                 }
 
                 /*
-                 * Resetting is slow, so skip if the subspace iteration has
-                 * only a single inner loop.
+                 * Resetting is slow, so skip if the subspace iteration is
```

Comment:
Yes, precisely. And no, I don't know a nice word for that property, since its not contiguous....